appender/influxdb/src/main/java/org/apache/karaf/decanter/appender/influxdb/InfluxDbAppender.java

```diff
@@ -120,16 +121,23 @@ public void handleEvent(Event event) {
             if (event.getProperty("type") != null) {
                 type = (String) event.getProperty("type");
             }
-            Map<String, Object> data = new HashMap<>();
+
+            Map<String, Object> eventFields = new HashMap<>();
+            Map<String, String> eventTags = new HashMap<>();
+            eventTags.putAll(globalTags);
+
             for (String propertyName : event.getPropertyNames()) {
                 Object propertyValue = event.getProperty(propertyName);
```

Comment:
Maybe I misunderstood the config `tag.` prefix, but I think it's to "detect" property that should consider as tags.
So, basically, something like:
```
if (tagsNames.contains(propertyName) && propertyValue != null && (propertyValue instanceof String)) {
  eventTags.put(propertyName, (String) propertyValue);
}
```
Else, I don't see the purpose of `eventTags.putAll(globalTags)`.

Thoughts ?

Reply:
actually, at first the "tag." prefix was used for global tags.
the purpose of "eventTags.putAll(globalTags)"
is to merge the global tags and event tags.